templates/account/is-paying-disconnect-button.php

```diff
@@ -0,0 +1,55 @@
+<?php
+    /**
+     * @package     Freemius
+     * @copyright   Copyright (c) 2016, Freemius, Inc.
+     * @license     https://www.gnu.org/licenses/gpl-3.0.html GNU General Public License Version 3
+     * @since       2.5.0
+     */
+
+    if ( ! defined( 'ABSPATH' ) ) {
+        exit;
+    }
+
+    /**
+     * @var array $VARS
+     * @var Freemius $fs
+     */
+    $fs                 = $VARS['freemius'];
+    $premium_plan_title = $VARS['premium_plan_title'];
+    $slug               = $VARS['slug'];
+
+    $button_id = "fs-disconnect-button-{$fs->get_id()}";
+
+    $product_type = $fs->get_module_label( true );
+
+    $message = $premium_plan_title ?
+        sprintf( fs_text_inline( 'Your %s plan is not active on this site at the moment and you can activate it on any other site. Only disconnect if you no longer want to use this %s anymore. If you wish to stop the recurring payment of the %s plan instead, then kindly click the CANCEL button, activate the %s plan and click on the "Downgrade" button to downgrade your account. Are you sure you would like to continue with this disconnection?', 'disconnect-x-confirm', $slug ), $premium_plan_title, $product_type, $premium_plan_title, $premium_plan_title ) :
```

Comment:
@xiahengchen Please use positioned placeholder. With translators comments (also valid for the next line).

```php
/** translators: %1$s is replaced by the premium plan name, %2$s is replaced by the type of the module */
sprintf( fs_text_inline( 'Your %1$s plan is not active on this site at the moment and you can activate it on any other site. Only disconnect if you no longer want to use this %2$s anymore. If you wish to stop the recurring payment of the %1$s plan instead, then kindly click the CANCEL button, activate the %1$s plan and click on the "Downgrade" button to downgrade your account. Are you sure you would like to continue with this disconnection?', 'disconnect-x-confirm', $slug ), $premium_plan_title, $product_type )
```